lib/schema.js

```diff
@@ -0,0 +1,72 @@
+const Joi = require('@hapi/joi')
+
+const fields = {
+  daysUntilStale: Joi.number()
+    .description('Number of days of inactivity before an Issue or Pull Request becomes stale'),
+
+  daysUntilClose: Joi.alternatives().try(Joi.number(), Joi.boolean().only(false))
+    .error(() => '"daysUntilClose" must be a number or false')
+    .description('Number of days of inactivity before a stale Issue or Pull Request is closed. If disabled, issues still need to be closed manually, but will remain marked as stale.'),
+
+  onlyLabels: Joi.alternatives().try(Joi.any().valid(null), Joi.array().single())
+    .description('Only issues or pull requests with all of these labels are checked for staleness. Set to `[]` to disable'),
+
+  exemptLabels: Joi.alternatives().try(Joi.any().valid(null), Joi.array().single())
+    .description('Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable'),
+
+  exemptProjects: Joi.boolean()
+    .description('Set to true to ignore issues in a project (defaults to false)'),
+
+  exemptMilestones: Joi.boolean()
+    .description('Set to true to ignore issues in a milestone (defaults to false)'),
+
+  exemptAssignees: Joi.boolean()
+    .description('Set to true to ignore issues with an assignee (defaults to false)'),
+
+  staleLabel: Joi.string()
+    .description('Label to use when marking as stale'),
+
+  markComment: Joi.alternatives().try(Joi.string(), Joi.any().only(false))
+    .error(() => '"markComment" must be a string or false')
+    .description('Comment to post when marking as stale. Set to `false` to disable'),
+
+  unmarkComment: Joi.alternatives().try(Joi.string(), Joi.boolean().only(false))
+    .error(() => '"unmarkComment" must be a string or false')
+    .description('Comment to post when removing the stale label. Set to `false` to disable'),
+
+  closeComment: Joi.alternatives().try(Joi.string(), Joi.boolean().only(false))
+    .error(() => '"closeComment" must be a string or false')
+    .description('Comment to post when closing a stale Issue or Pull Request. Set to `false` to disable'),
+
+  limitPerRun: Joi.number().integer().min(1).max(30)
+    .error(() => '"limitPerRun" must be an integer between 1 and 30')
+    .description('Limit the number of actions per hour, from 1-30. Default is 30')
+}
+
+const schema = Joi.object().keys({
+  daysUntilStale: fields.daysUntilStale.default(60),
+  daysUntilClose: fields.daysUntilClose.default(7),
+  onlyLabels: fields.onlyLabels.default([]),
+  exemptLabels: fields.exemptLabels.default(['pinned', 'security']),
+  exemptProjects: fields.exemptProjects.default(false),
+  exemptMilestones: fields.exemptMilestones.default(false),
+  exemptAssignees: fields.exemptMilestones.default(false),
+  staleLabel: fields.staleLabel.default('wontfix'),
```

Comment:
There must be a better label than `wontfix` which sounds rather abrupt and off putting. Can we use `no-issue-activity` or something that clearly indicates a stale issue rather than `wontfix` as the default?

Reply:
Yes, I think no-issue-activity looks good, I will update that right away.